test/unfurl-to-attachment.test.js

```diff
@@ -19,9 +19,9 @@ const urls = {
     text: 'node-github - node library to access the GitHub API',
     thumb_url: 'https://avatars0.githubusercontent.com/u/3430433?s=400&v=4'
   },
-
+  // Domain that returns relative author_icon
   'https://www.npmjs.com/package/unfurl.js': {
-    author_icon: '/static/images/touch-icons/favicon-16x16.png',
+    author_icon: 'https://www.npmjs.com/static/images/touch-icons/favicon-16x16.png',
```

Comment:
Isn't the whole point of this PR to do this automatically? I'd love to see a separate test for this use case rather than editing an existing one.

Reply:
@hiimbex This url `https://www.npmjs.com/package/unfurl.js`, will return author icon's url as `/static/images/touch-icons/favicon-16x16.png` and that appears as broken image but after i added the functionality to handle this, the result will come out to be `https://www.npmjs.com/static/images/touch-icons/favicon-16x16.png` . So to make the test pass, i made it absolute url that is actually returned by `unfurl-to-attachment.js` file.

Reply:
Should i make the test more descriptive?